.github/workflows/canary-integration-test.yml

```diff
@@ -309,11 +330,18 @@ jobs:
         name: pvc-db-wal
         path: test
 
-    - name: setup tmate session for debugging
-      if: failure()
+    - name: setup tmate session for debugging when event is PR
+      if: failure() && github.event_name == 'pull_request'
```

Comment:
is there no `else` statement in github actions?

Reply:
no, I have not seen else in doc.